libs/elements/layout/src/lib/website-header/website-header.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-website-header',
+  templateUrl: './website-header.component.html',
+  styleUrls: ['./website-header.component.scss'],
+})
+export class WebsiteHeaderComponent implements OnInit {
+  isNavLinks!: boolean;
+  ngOnInit(): void {
+    this.isNavLinks = false;
+  }
+  setVisibility() {
+    this.isNavLinks = !this.isNavLinks;
+  }
+}
```

Comment:
Create a components folder inside lib and move this component into it. There'll be multiple elements inside this lib in different folders